### PR TITLE
Factor out specs

### PR DESCRIPTION
**Pull request by oskarth**
Separate out specs

@nolash

### CODE REVIEW

**Comment by nolash** [comment]
Ok great. Would it be easier to maintain and keep overview if these are in the _same_ file, and with a checklist for each individual spec?

**Comment by nolash** [comment]
I see; together with the overview it makes sense to have individual ones.

File: user_bzz_read_spec.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+name: Swarm BZZ read spec
+about: Swarm BZZ read spec
+title: Swarm BZZ read spec
```
Comment (nolash) [inline user_bzz_read_spec.md:4]: Isn't BZZ light = BZZ read + BZZ readwrite?

File: user_pss_adaptive_server_spec.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+---
+name: Swarm PSS adaptive server spec
+about: Swarm PSS adaptive server spec
+title: Swarm PSS adaptive server spec
```
Comment (nolash) [inline user_pss_adaptive_server_spec.md:4]: I'm not sure we need special server-side considerations for `pss`.

**Review by nolash — CHANGES_REQUESTED** [review]

File: user_bzz_read_spec.md (hunk 1 above)
Comment (oskarth) [reply user_bzz_read_spec.md:4]: What's BZZ full in that case? BZZ light is more about (not) syncing as far as I understood it.

File: user_pss_adaptive_server_spec.md (hunk 2 above)
Comment (oskarth) [reply user_pss_adaptive_server_spec.md:4]: Ok, so perhaps a general `user_adaptive_server_spec` would suffice?

**Comment by oskarth** [comment]
PR no longer relevant, closing

**Pull request closed without merging**